nodestream/databases/database_connector.py

```diff
@@ -15,7 +15,7 @@ class DatabaseConnector(ABC, Pluggable):
 
     @classmethod
     def from_database_args(
-        cls, database: str = "neo4j", **database_args
+        cls, database: str = "null", **database_args
```

Comment:
This is is just so the null connector is used instead of a default that may or may not be installed.